OJP/OJP_Trips.xsd

```diff
@@ -1581,11 +1613,7 @@ Can only be used when both participants recognise the same schedule version. If
 					<xs:documentation>System reference to use for the refinement. If not specified the origin systems of each leg are used for the refinement.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
-			<xs:element name="UseRealtimeData" type="UseRealtimeDataEnumeration" default="full" minOccurs="0">
```

Comment:
Question: where did UseRealtimeData move to?

Reply:
I think it was removed by mistake